patches/chrome-browser-android-browsing_data-browsing_data_bridge.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/android/browsing_data/browsing_data_bridge.cc b/chrome/browser/android/browsing_data/browsing_data_bridge.cc
+index cd5bfb077e93e2e9904ebf4287860c966776b869..144150ee92a304e990fa3de83ed33dc3480eb91c 100644
+--- a/chrome/browser/android/browsing_data/browsing_data_bridge.cc
++++ b/chrome/browser/android/browsing_data/browsing_data_bridge.cc
+@@ -123,6 +123,7 @@ static void JNI_BrowsingDataBridge_ClearBrowsingData(
+             ChromeBrowsingDataRemoverDelegate::DATA_TYPE_CONTENT_SETTINGS;
+         break;
+       case browsing_data::BrowsingDataType::DOWNLOADS:
++        remove_mask |= BrowsingDataRemover::DATA_TYPE_DOWNLOADS; break;
+       case browsing_data::BrowsingDataType::HOSTED_APPS_DATA:
+         // Only implemented on Desktop.
```

Comment:
I'm a bit confused here because this says that downloads and hosted app data deletion is only implemented on desktop. Are you sure setting this mask actually works and why isn't it implemented on chromium in the first place?